.github/workflows/test.yml

```diff
@@ -24,3 +24,19 @@ jobs:
       command: |
         mise run install
         mise run ci
+  build:
```

Comment:
We can remove this now no?

Reply:
@uptickmetachu we don't want test builds ?

Reply:
I guess if we have the lifecycle policies. 

I don't think its entirely too neccessary but it doesn't hurt too much.